src/routes/FindItFixIt.js

```diff
@@ -0,0 +1,275 @@
+/* eslint-disable */
```

Comment:
Please do not disable lint

Reply:
Do you have prettier configuration to match your eslint configuration?

Reply:
I don't unfortunately